src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1347,21 +1347,27 @@ class DocBaseViewer extends BaseViewer {
 
         let metricName;
         let eventName;
+        let style;
         if (!this.thumbnailsSidebar.isOpen) {
             this.rootEl.classList.remove(CLASS_BOX_PREVIEW_THUMBNAILS_OPEN);
             metricName = USER_DOCUMENT_THUMBNAIL_EVENTS.CLOSE;
             eventName = 'thumbnailsClose';
+            style = 'display: none';
         } else {
             this.rootEl.classList.add(CLASS_BOX_PREVIEW_THUMBNAILS_OPEN);
             metricName = USER_DOCUMENT_THUMBNAIL_EVENTS.OPEN;
             eventName = 'thumbnailsOpen';
+            style = '';
         }
 
         this.emitMetric({ name: metricName, data: pagesCount });
         this.emit(eventName);
 
         // Resize after the CSS animation to toggle the sidebar is complete
-        setTimeout(() => this.resize(), THUMBNAILS_SIDEBAR_TRANSITION_TIME);
+        setTimeout(() => {
+            this.resize();
+            this.thumbnailsSidebarEl.style = style;
```

Comment:
Seems like setting `style` directly as a string is discouraged (https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style#Setting_styles).

Also, does this affect the animation transition at all?

Reply:
It does appear to affect the hidden -> visible transition. I'll update it to remove the `display: none` immediately rather than via setTimeout for that case. Good catch!